interfaces/acados_template/acados_template/c_templates/acados_solver.in.c

```diff
@@ -29,6 +29,7 @@
  * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.;
+>>>>>>> 74cd154c805e926ec66fe3dfd69bb542f3771c87
```

Comment:
oops, seems like some merge went wrong.

Edit: but you said this folder is `inja` and will be removed soon anyway

Reply:
good catch (addressed here 104604a). Note: this is the Jinja folder :)